cmd/statreceiver/filter.go

```diff
@@ -5,29 +5,56 @@ package main
 
 import (
 	"regexp"
+	"sync"
 	"time"
+
+	"github.com/zeebo/admission/admproto"
 )
 
-// PacketFilter is used during Packet parsing to determine if the Packet should
-// continue to be parsed.
+// PacketFilter inspects a packet header to determine if it should be passed
+// through
 type PacketFilter struct {
 	application *regexp.Regexp
 	instance    *regexp.Regexp
+	dest        PacketDest
+	scratch     sync.Pool
 }
 
-// NewPacketFilter creates a PacketFilter. It takes an application regular
-// expression and an instance regular expression. If the regular expression
-// is matched, the packet will be parsed.
-func NewPacketFilter(applicationRegex, instanceRegex string) *PacketFilter {
+// NewPacketFilter creates a PacketFilter. It takes a packet destination,
+// an application regular expression, and an instance regular expression.
+// If the regular expression is matched, the packet will be passed through.
+func NewPacketFilter(applicationRegex, instanceRegex string, dest PacketDest) *PacketFilter {
 	return &PacketFilter{
 		application: regexp.MustCompile(applicationRegex),
 		instance:    regexp.MustCompile(instanceRegex),
+		dest:        dest,
+		scratch: sync.Pool{
+			New: func() interface{} {
+				var x [10 * kb]byte
```

Comment:
I agree with this comment, using `memory.KB` is more readable and would be consistent throughout the codebase
https://github.com/storj/storj/pull/636#discussion_r234138547

Reply:
fair